### PR TITLE
Fixes openSUSE Leap 15.1 box name

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # sesci
 
-Jenkins jobs configs can be found in `jenkins/jjb` for jenkins job builder or `jenkins/job-dsl` for Jenkins Job DSL plugin.
+Jenkins jobs configs can be found in `jjb/` for jenkins job builder or `jenkins/job-dsl` for Jenkins Job DSL plugin.
```
Comment (kshtsk) [inline README.md]: the '/' is extra here, could you please remove it?

File: jjb/vagrant-ceph.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@
             name: BOX
             type: user-defined
             values:
-                - 'sub0/leap151'
+                - 'virt-appl/openSUSE-Leap-15.1'
```
Comment (kshtsk) [inline jjb/vagrant-ceph.yaml:26]: @colder-is-better could you please confirm this change is appropriate.

File: README.md (hunk 1 above)
Comment (toabctl) [reply README.md]: Done

File: jjb/vagrant-ceph.yaml (hunk 2 above)
Comment (colder-is-better) [reply jjb/vagrant-ceph.yaml:26]: @kshtsk Yes, that is the name of the box in https://github.com/openSUSE/vagrant-ceph/blob/master/config.yml (The "virt-appl" prefix implies that the Leap 15.1 box comes from the Virtualization Appliances repo.)

**Review by kshtsk — APPROVED** [review]

**Pull request merged**